src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -578,12 +669,15 @@ class HookUtils:
             result = self.hook_events[event.type][
                 'handler'](event, cgroup, jobutil, *args)
             return result
-        else:
-            pbs.logmsg(pbs.EVENT_DEBUG2,
-                       "%s: %s event not handled by this hook." %
-                       (caller_name(), self.event_name(event.type)))
+        pbs.logmsg(pbs.EVENT_DEBUG2,
+                   "%s: %s event not handled by this hook." %
+                   (caller_name(), self.event_name(event.type)))
+        return False
 
-    def __execjob_begin_handler(self, e, cgroup, jobutil):
+    def __execjob_begin_handler(self, event, cgroup, jobutil):
```

Comment:
Since they're really not designed to be subclasses, the mangling isn't necessary. Changed them to single underscores since they are still private methods.